packages/docs/getting-started/running-a-validator.md

```diff
@@ -117,177 +133,170 @@ In all the commands we are going to see the `CELO_IMAGE` variable to refer to th
 
 ```bash
-mkdir -p celo-data-dir/proxy celo-data-dir/validator celo-data-dir/attestations
-cd celo-data-dir
+# On your local machine
+mkdir -p celo-data-dir/accounts
+cd celo-data-dir/accounts
+docker run -v $PWD/accounts:/root/.celo --entrypoint /bin/sh -it $CELO_IMAGE -c "geth account new"
+docker run -v $PWD/accounts:/root/.celo --entrypoint /bin/sh -it $CELO_IMAGE -c "geth account new"
 ```
 
-It is worth noting that we create all these accounts on the same machine right now, but you are expected to use these accounts on the services on separate machines. We are going to need to create 4 accounts, 2 for the Validator, 1 for the Proxy and the last one for the Attestation Service. You can generate their addresses using the below commands if you don’t already have them. If you already have some accounts, you can skip this step.
-
-To create the accounts needed, run the following commands. The first two blocks create the accounts for the Validator, the third one for the Proxy, and the last one for the attestation service. Also we save these addresses to environment variables, so that you can reference it later (don't include the braces):
+This should generate two accounts in your current directory and print them out, set them in an environment variables:
 
 ```bash
-docker run -v $PWD/validator:/root/.celo --entrypoint /bin/sh -it $CELO_IMAGE -c "geth account new"
+# On your local machine
 export CELO_VALIDATOR_GROUP_ADDRESS=<YOUR-VALIDATOR-GROUP-ADDRESS>
-
-docker run -v $PWD/validator:/root/.celo --entrypoint /bin/sh -it $CELO_IMAGE -c "geth account new"
 export CELO_VALIDATOR_ADDRESS=<YOUR-VALIDATOR-ADDRESS>
-
-docker run -v $PWD/proxy:/root/.celo --entrypoint /bin/sh -it $CELO_IMAGE -c "geth account new"
-export CELO_PROXY_ADDRESS=<YOUR-PROXY-ADDRESS>
-
-docker run -v $PWD/attestations:/root/.celo --entrypoint /bin/sh -it $CELO_IMAGE -c "geth account new"
-export ATTESTATION_SIGNER_ADDRESS=<YOUR-ATTESTATION_SIGNER-ADDRESS>
 ```
 
-Those commands will prompt you for a passphrase, ask you to confirm it, and then will output your account address: `Address: {<YOUR-ACCOUNT-ADDRESS>}`
-
-{% hint style="danger" %}
-**Warning**: There is a known issue `(Fatal: Failed to read passphrase: liner: function not supported in this terminal” rather than just failing)` running geth inside Docker that happens eventually. So if that command fails, please try again, and if it continues to fail after several attempts, check this page [this page](https://forum.celo.org/t/setting-up-a-validator-faq/90).
-{% endhint %}
-
-In order to register the Validator later on, generate a "proof of possession" - a signature proving you know your Validator's BLS private key. Run this command to generate this "proof-of-possession", which consists of a the BLS public key and a signature:
+Before we can run the node with these accounts, we just have to setup the genesis file and setup some static nodes:
```

Comment:
Next, we'll run a node on your local machine so that we can use these accounts to lock Celo Gold and authorize the keys needed to run your validator. To do this, we need to run the following commands, which fetch the genesis block and a list of other nodes in the network to connect to.